backend/driver.pm

```diff
@@ -144,6 +144,16 @@ sub send_key {
     return $self->_send_json({cmd => 'send_key', arguments => {key => $key}});
 }
 
+sub hold_key {
```

Comment:
you don't need these if you call the backend function as hold_key({key => $key}). The send_key function is historic